__tests__/integration-tests.test.js

```diff
@@ -33,6 +33,36 @@ describe("GET /api/topics tests", () => {
     })
 });
 
+describe("GET /api/articles tests", () => {
+        test('responds with array of objects with the correct keys. The array has the correct length and the accompanying code should be status 200', () => {
+            return request(app)
+            .get("/api/articles")
+            .expect(200)
+            .then(( { body }) => {
+                
+                const output = Array.isArray(body.articles)
+                const expected = true
+                expect(output).toBe(expected)
```

Comment:
This is a nice expect but becomes redundant due to the following expect statement of .toHaveLength and using a .forEach as these indirectly also test that the articles is an array.